frontend/Gemfile.lock

```diff
@@ -207,7 +213,7 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  fastlane (~> 2.194)
+  fastlane (>= 2.214.0)
```

Comment:
@ztefanie is it no issue with new fastlane version?

Reply:
I guess not but not sure though